### ISSUE TITLE
Color

### ISSUE THREAD

**Issue by SumZer0-git** [Matrixchung/EDAutopilot#12]
I use the default Colors in ED.   Was thinking to load all the template images and convert to gray scale
image = cv2.cvtColor(image, cv2.COLOR_GRAY2RGB)

Then when grabbing screen captures, to convert that to gray scale also and attempt template matching that way.

This way a user does not have to adjust their ED color scheme.

Thoughts?
(also nice work on the key bindings)

**Comment by SumZer0-git** [comment]
yes, I ran into the console being washed out by the Star.   In doing jumping between systems, if you Pitch up over the Star then the Star is beneath the Ship and does not wash out the compass.    However, there is still cases when maneuvering to the target where the Star shines into the cockpit (if the next Target is to the left or right of the Star and not behind it)

**Comment by Matrixchung** [comment]
That's where the problem is.
In normal cases, we will do `sunAvoiding` between jumps, but in some rare situations when we perform in-system supercruise, there's a chance of running into bright sunlight.

**Comment by Matrixchung** [comment]
I am wondering that if the problem is unavoidable, should we pitch up again for some secs then?
Or is there any solution to it

**Comment by SumZer0-git** [comment]
yep, not sure easy solution.   Thanks for feedback.
